### PR TITLE
Generate codeowners

### LINKED ISSUE #86: Add codeowners
To get pinged and have actual owners who keep up with the software that the modules are taking advantage of.

### PR DESCRIPTION
**Pull request by Emiller88**
Closes #86

### CODE REVIEW

**Comment by maxulysse** [comment]
Can we add as well the modules' contributors to codeowners?

**Comment by Emiller88** [comment]
Sure, that might get really hetic. I'll make another PR switch authors to `maintainers` and then we can add them as we go.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,3 +2,2118 @@
...
+modules/nf-core/mash/sketch/** @thanhleviet
+modules/nf-core/iphop/predict/** @CarsonJM
+modules/nf-core/mash/dist/** @rpetit3
+modules/nf-core/abricate/summary/** @rpetit3
+modules/nf-core/seroba/run/** @rpetit3
+modules/nf-core/mash/screen/** @mahesh-panchal
+modules/nf-core/sequencetools/pileupcaller/** @TCLamnidis
+modules/nf-core/abricate/run/** @rpetit3
+modules/nf-core/fastqc/** @drpatelh @grst @ewels @FelixKrueger
+modules/nf-core/sgdemux/** Samantha White: @samfulcrum
```
Comment (grst) [inline .github/CODEOWNERS:232]: @Emiller88 , what's going on in this line?

Comment (Emiller88) [reply .github/CODEOWNERS:232]: Good catch, I think just a typo with the format from @samfulcrum

**Pull request merged**